UnityProject/Assets/Scripts/Managers/CentComm.cs

```diff
@@ -78,11 +79,14 @@ public void UpdateStatusDisplay(StatusDisplayChannel channel, string text)
 	private string AntagInitialUpdate =
 		"Enemy communication intercepted. Security level elevated.";
 
-	void Start()
+	public InitialisationSystems Subsystem => InitialisationSystems.CentComm;
+
+	void IInitialise.Initialise()
 	{
 		paperPrefab = Resources.Load<GameObject>("Paper");
```

Comment:
this should probably be a GameObject field rather than magical string loading.

Reply:
I know, not part of this PR. Just commenting for later.

Reply:
good catch